src/fcmp_pp/curve_trees.h

```diff
@@ -129,23 +132,26 @@ struct OutputContext final
     inline void serialize(Archive &a, const unsigned int ver)
     {
         a & output_id;
+        a & torsion_checked;
         a & output_pair;
     }
 
     BEGIN_SERIALIZE_OBJECT()
         FIELD(output_id)
+        FIELD(torsion_checked)
         FIELD(output_pair)
     END_SERIALIZE()
 
     BEGIN_KV_SERIALIZE_MAP()
         KV_SERIALIZE(output_id)
+        KV_SERIALIZE(torsion_checked)
         KV_SERIALIZE(output_pair)
     END_KV_SERIALIZE_MAP()
 };
 #pragma pack(pop)
 
 static_assert(sizeof(OutputPair)    == (32+32),   "db expects 64 bytes for output pairs");
-static_assert(sizeof(OutputContext) == (8+32+32), "db expects 72 bytes for output context");
+static_assert(sizeof(OutputContext) == (8+1+32+32), "db expects 73 bytes for output context");
```

Comment:
This is very nit picky, but the size of a boolean is implementation-defined, and not guaranteed to be 1 on every system: https://stackoverflow.com/questions/4897844/is-sizeofbool-defined-in-the-c-language-standard/4897859#4897859

Reply:
That's interesting, so either I use a different type here or use `sizeof(bool)` in place of 1. Maybe a uint8_t would keep things simple here